### PR TITLE
download: skoala version updated to v0.33.3

### PR DESCRIPTION
**Pull request by daocloudpublic**
download: skoala version updated to v0.33.3

### CODE REVIEW

File: docs/zh/docs/download/modules/skoala.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ hide:
 
 | 版本 | 架构 | 文件大小 | 安装包 |  校验文件 | 更新日期   |
 | --- | ----- | ----- | ------ | ------- | --------- |
+| [v0.33.3](../../skoala/intro/release-notes.md) | AMD 64 | 1.60GB | [:arrow_down: skoala_v0.33.3_amd64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/skoala_v0.33.3_amd64.tar) | [:arrow_down: skoala_v0.33.3_amd64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/skoala_v0.33.3_amd64_checksum.sha512sum) | 2024-03-01 |
```
Comment (windsonsea) [inline docs/zh/docs/download/modules/skoala.md:17]: 咦，版本回退了吗？上一个是0.34哇

Comment (wilsonwu) [reply docs/zh/docs/download/modules/skoala.md:17]: It is a hotfix version.

**Comment by wilsonwu** [comment]
@windsonsea please close this PR 0.33.3 has problem, will release 0.33.4 soon to instead.

**Comment by windsonsea** [comment]
ok, will close it soon.

**Pull request closed without merging**